src/ad-hoc-visualizations/needs-review/visualization.tsx

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { AdHocTestkeys } from 'common/configs/adhoc-test-keys';
+import { TestMode } from 'common/configs/test-mode';
+import { VisualizationConfiguration } from 'common/configs/visualization-configuration';
+import { generateUID } from 'common/uid-generator';
+import { AdhocIssuesTestView } from 'DetailsView/components/adhoc-issues-test-view';
+import { VisualizationInstanceProcessor } from 'injected/visualization-instance-processor';
+import * as React from 'react';
+
+export const NeedsReviewAdHocVisualization: VisualizationConfiguration = {
+    key: AdHocTestkeys.NeedsReview,
+    testMode: TestMode.Adhoc,
+    getTestView: props => <AdhocIssuesTestView {...props} />,
+    getStoreData: data => data.adhoc.needsReview,
+    enableTest: (data, _) => (data.enabled = true),
+    disableTest: data => (data.enabled = false),
+    getTestStatus: data => data.enabled,
+    displayableData: {
+        title: 'Needs Review',
+        enableMessage: 'Running needs review checks...',
+        toggleLabel: 'Show areas needing review',
+        linkToDetailsViewText: 'List view and filtering',
+    },
+    chromeCommand: '',
```

Comment:
I know that the other ad-hoc visualizations have a chromeCommand; will this be added later in the feature?

Reply:
Actually resolving b/c remembered that's most likely reserved for implementation. Nevermind!